src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialJsPage.java

```diff
@@ -40,7 +45,7 @@ public SpecialJsPage open(String articleTitle) {
         return this;
     }
 
-    public String getScriptCoentent() {
+    public String getScriptContent() {
```

Comment:
What is the "script" and why does it have any content ?
I'd expect article / profile / section to have content, but i can't get what is a "script" content